tos.md

```diff
@@ -36,6 +36,20 @@ Permission is granted to use the services and functionality on Gradebook's websi
 
 This license shall automatically terminate if you violate any of these restrictions and may be terminated by Gradebook at any time. Upon terminating your viewing of these materials or upon the termination of this license, you must destroy any proprietary materials in your possession regardless of format.
 
+## Termination of Services
+
+Gradebook reserves the right to terminate your account under certain conditions, including but not limited to:
+
+ - Account inactivity (though you will be given reasonable notice)
+ 
+ - Abuse
+ 
+ - Requesting account deletion through the Account page of the Application
+ 
+ - Requesting account deletion via email
```

Comment:
```suggestion
 - Requesting account deletion via email to contact@gradebook.app
```

Reply:
I think leaving it at as-is is fine. Wdyt @ramsays?

Reply:
I would say leave as-is.